milestone-1-survey.md

```diff
@@ -0,0 +1,92 @@
+- *Dynamic Consistency Boundary*: We are breaking free from the aggregate concept. Entities now allow a flexible
+  boundaries, making it possible to combine events from different tags. This greatly reduces the need to use sagas, and
+  makes code flexible for whatever requirements pop up in the future.
+
+We plan to release at least two more milestones releases before the final release of Axon Framework 5. This milestone
+release does not contain, for example:
+
+- Spring Boot support
+- Redesigned event processors
+- TODO: Base on roadmap?
+
+## Try it out
+
+We encourage you to try out the milestone release and share your feedback with us.
+You can follow the instructions in
+the [Getting Started with Axon Framework 5 guide](https://docs.axoniq.io/axon-framework/todo-fill-in-url) to get
+familiar with the new features and changes. After that, please share your feedback with us with the template down below.
+
+## Feedback template
+
+Please fill in the following template to share your feedback with us.
+
+1. How would you rate the overall experience of using Axon Framework 5 M1?
+    - [ ] 1: Very poor
+    - [ ] 2: Poor
+    - [ ] 3: Average
+    - [ ] 4: Good
+    - [ ] 5: Excellent
+2. What did you like most about Axon Framework 5 M1?
+    - [ ] Asynchronous core
+    - [ ] Simplified configuration
+    - [ ] Dynamic consistency boundary
+    - [ ] Other: _______________
+3. Do you think Axon Framework 5 improves the developer experience compared to Axon Framework 4?
+    - [ ] Yes, definitely
+    - [ ] Probably
+    - [ ] Maybe
+    - [ ] Not sure
+    - [ ] Probably not
+    - [ ] Definitely not
+4. How long did it take you to get started with Axon Framework 5 M1?
+    - [ ] Less than 1 hour
+    - [ ] 1-2 hours
+    - [ ] 2-4 hours
+    - [ ] More than 4 hours
+5. How intuitive did you find the new configuration system?
+    - [ ] Very intuitive
+    - [ ] Somewhat intuitive
+    - [ ] Neutral
+    - [ ] Somewhat unintuitive
+    - [ ] Very unintuitive
+6. Did you try out the new dynamic consistency boundary feature? If so, how would you rate it?
+    - [ ] Yes, I loved it!
+    - [ ] Yes, it was okay
+    - [ ] Yes, but I didn't find it useful
+    - [ ] No, I didn't try it out
+7. Would you consider using Axon Framework 5 in a production project once it's released?
+    - [ ] Yes, definitely
+    - [ ] Probably
+    - [ ] Maybe
+    - [ ] Not sure
+    - [ ] Probably not
+8. Do you have any suggestions to improve the new features in Axon Framework 5 M1?
+    - [ ] Yes: ________________
+    - [ ] No, Everything is perfect!
+
+Lastly, please share any additional feedback or comments you have about Axon Framework 5 M1:
+____________________________________________________________________________________
+____________________________________________________________________________________
+____________________________________________________________________________________
+
```

Comment:
I think it'd be nice to give such option for every question. In case of 6 it may be worth to encourage people to write what's their DX regarding EventCriteria.